start_rc.py

```diff
@@ -64,18 +65,28 @@ def download_if_necessary(repo, artifact_id, version, is_test_artifact=False):
 def start_rc(stdout=None, stderr=None):
     artifacts = []
 
-    rc = download_if_necessary(RC_REPO, "hazelcast-remote-controller", RC_VERSION)
-    tests = download_if_necessary(REPO, "hazelcast", SERVER_VERSION, True)
-    sql = download_if_necessary(REPO, "hazelcast-sql", SERVER_VERSION)
+    rc = download_if_necessary(RC_REPO, HAZELCAST_GROUP, "hazelcast-remote-controller", RC_VERSION)
+    tests = download_if_necessary(REPO, HAZELCAST_GROUP, "hazelcast", SERVER_VERSION, True)
+    sql = download_if_necessary(REPO, HAZELCAST_GROUP, "hazelcast-sql", SERVER_VERSION)
 
     artifacts.extend([rc, tests, sql])
 
     enterprise_key = os.environ.get("HAZELCAST_ENTERPRISE_KEY", None)
 
     if enterprise_key:
-        server = download_if_necessary(ENTERPRISE_REPO, "hazelcast-enterprise", SERVER_VERSION)
+        server = download_if_necessary(
+            ENTERPRISE_REPO, HAZELCAST_GROUP, "hazelcast-enterprise", SERVER_VERSION
+        )
+        vector = download_if_necessary(
+            ENTERPRISE_REPO, HAZELCAST_GROUP, "hazelcast-enterprise-vector", SERVER_VERSION
+        )
+        artifacts.append(vector)
+        jvector = download_if_necessary(RELEASE_REPO, "io.github.jbellis", "jvector", "2.0.5")
+        artifacts.append(jvector)
+        math3 = download_if_necessary(RELEASE_REPO, "org.apache.commons", "commons-math3", "3.6.1")
```

Comment:
I'd say let's put these versions at the top of the file, next to SERVER_VERSION &
RC_VERSION.

Why do we need these jars though? Should not "hazelcast-enterprise-vector" include them(any dependency needed for vectors) already? If you are going to shade those dependencies into hazelcast-enterprise-vector soon, I'd add a todo comment here about removing these dependencies from here because it seems this pr will be open for a while until 5.4 is released

Reply:
Currently `hazelcast-enterprise-vector` jar doesn't include those dependencies.
But eventually it will, so we can just delete those lines when that happens.

Reply:
https://github.com/hazelcast/hazelcast-python-client/pull/679